.changeset/moody-bugs-do.md

```diff
@@ -0,0 +1,5 @@
+---
+"@digdir/designsystemet-css": patch
+---
+
+fixed the typo in `--dsc-details-border-block-wdith`. The old variable is kept for now, so there are no breaking changes
```

Comment:
```suggestion
**Details**: Added `--dsc-details-border-block-width` to fix typo in `--dsc-details-border-block-wdith`.
The old variable is kept as is, so there are no breaking changes.
```
Not a fan of my own wording here, but lets at least show the new variable as well

Reply:
Yes better 👍